### PR TITLE
Add translations

### PR DESCRIPTION
**Pull request by EsaMakinen**

### CODE REVIEW

File: frontend/embed/v1/src/translations.json [hunk 1]
```diff
@@ -4,13 +4,114 @@
...
+            oireita vai ei.",
+        "start_info_2": "Kertomalla tilanteestasi – myös terveenä – voit auttaa mediaa kertomaan paremmin siitä, miten virus leviää.
+            Oiretutkaan tallennettuja tietoja ei voi yhdistää vastaajaan. Vastauksia julkaistaan yhteenvetona ja
+            yksittäisinä vastauksina. Kysely on suunniteltu yhteistyössä Helsingin yliopiston kansanterveystieteen
+            osaston kanssa.",
+        "start_info_3": "Palvelua ei ole tarkoitettu yksittäisen henkilön opastamiseksi terveyteensä liittyen tai ohjaamiseksi
+            terveydenhuoltoon.",
+        "privacy_readmore": "Lue lisää tietosuojasta täältä.",
+        "start_survey": "Aloita",
+        "close_survey":" "Sulje kysely",
```
Comment (sjarva) [inline frontend/embed/v1/src/translations.json:17]: Here's an extra quote that is messing up the rest of the file :/

File: frontend/embed/v1/src/translations.json [hunk 2]
```diff
@@ -4,13 +4,114 @@
...
+        "page_title": "“Let us know if you are experiencing symptoms or not, and help us determine how COVID-19 spreads.",
+        "start_info_1": "Finnish media providers are requesting information from their readers to let them know if they are currently experiencing any potential coronavirus symptoms via the Oiretutka service.",
+        "start_info_2": "By sharing anonymous information about your situation- even if you are currently feeling completely healthy- you can help the media to cover the spread of the virus more effectively and accurately. Any responses provided via the Oiretutka service questionnaire are totally anonymous, and cannot be traced back to the respondent. The replies will be published both in a compiled format, as well as as standalone replies.The questionnaire has been created in collaboration with the department of Public Health Research at Helsinki University.",
+        "start_info_3": "This service is not meant to be used as an aid to self-diagnose symptoms, or as a guide to seek medical aid.",
+        "privacy_readmore": "More on privacy here.",
+        "start_survey": "Start",
+        "close_survey":" "Close the form",
```
Comment (sjarva) [inline frontend/embed/v1/src/translations.json:74]: Another extra quote

**Comment by EsaMakinen** [comment]
@sjarva I fixed those, please re-review

**Comment by mahdava** [comment]
Rebased :)

**Pull request merged**